presto-main/src/main/java/com/facebook/presto/operator/OrderByOperator.java

```diff
@@ -380,5 +380,6 @@ public void close()
     {
         pageIndex.clear();
         sortedPages = null;
+        spiller.ifPresent(Spiller::close);
```

Comment:
Should this be squashed with `Add Spill To Disk for ORDER BY`?

Reply:
I added this as a separate commit since it was a bug fixed in a later commit. However, that later commit also fixes the equivalent bug for window spilling. So, to make it explicit I just created a new commit. Can either squash this or leave it, no opinion on this.